packages/lingua-franca/src/templates/pages/index.tsx

```diff
@@ -29,25 +29,6 @@ const Index: React.FC<Props> = (props) => {
   const i = createInternational<typeof indexCopy>(useIntl())
   const Link = createIntlLink(props.pageContext.lang)
 
-
-  // useEffect(() => {
-  //   // NOOP on tiny devices where we need to re-orient the arrows.
-  //   if (window.innerWidth < 900) return
-
-  //   const adopt = document.getElementById("adopt-gradually-content")!
-  //   adopt.classList.remove("no-js")
-  //   adopt.classList.add("fancy-scroll")   
-
-  //   updateOnScroll(i)()
```

Comment:
I think that these being disabled might have to do with the menu bar not disappearing when scrolling on mobile. Instead of removing them, we should try to enable them.